save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt

```diff
@@ -269,7 +269,7 @@ class ExecutionController(private val executionService: ExecutionService,
             IllegalArgumentException("Can't rerun execution $id, because it does not exist")
         }
         if (!projectPermissionEvaluator.hasPermission(
-            authentication, execution.project, Permission.WRITE
```

Comment:
Why did you change this?
As we allow to submit execution only with `write` permissions, probably `rerun` should have the same restriction

Reply:
oh, changed by mistake